lib/ecto.ex

```diff
@@ -595,4 +595,21 @@ defmodule Ecto do
       true
     end
   end
+
+  @spec load(Ecto.Schema.t | map(), map() | Keyword.t | {list, list}) :: Ecto.Schema.t | map()
```

Comment:
FWIW even though `{fields, values}` is being used under the hood, it might make sense to NOT expose it to keep the interface tighter (callers would just do `Enum.zip`)